gbfs.md

```diff
@@ -273,9 +273,17 @@ Field Name | Required | Type | Defines
+\- `car_type` | Optional | Enum | The type of car. <br /><br />Current valid issues are: <br /><ul><li>`microcar` _(smallest size of car, engine smaller than 700cc, e.g Smart Fortwo)_ </li><li>`subcompact` _(sedan with interior volume index no larger than 99 cu ft, e.g. Kia Rio)_</li><li>`compact` _(sedan with interior volume index no larger than 109 cu ft, e.g. Hyundai Elantra)_ </li><li>`mid-size` _(sedan with interior volume index no larger than 119 u ft, e.g. Toyota Camry)_</li><li>`full_size` _(sedan with interior volume index larger than 120 cu ft, e.g. Tesla Model S)_</li><li>`station-wagon` _(large cargo area with rear hinged tailgate, e.g. Subaru Outback)_ </li><li>`suv` _(similar to station-wagon with a light truck frame, e.g. Honda Pilot)_ </li><li>`minivan` _(also known as multi-purpose vehicle or people mover, e.g. Dodge Grand Caravan)_ </li><li>`pickup truck` _(enclosed cab with open cargo area and tailgate, e.g. Chevrolet Ram)_</li><li>`other`
+\- `seating_capacity` | Optional | Non-negative integer | The number of riders the vehicle can legally accommodate.
+\- `share_type` | Optional | Enum | The sharing model of the vehicle.<br /><br />Current valid values are:<br /><br /><ul><li>`roundtrip` _(vehicle must be returned to pickup location)_</li><li>`one-way` _(vehicle can be returned to any station in the service area)_</li><li>`free_floating` _(vehicle can be returned anywhere in the service area)_
+\- `propulsion_type` | Yes | Enum | The primary propulsion type of the vehicle. <br /><br />Current valid values are:<br /><ul><li>`human` _(Pedal or foot propulsion)_</li><li>`electric_assist` _(Provides power only alongside human propulsion)_</li><li>`electric` _(Contains throttle mode with a battery-powered motor)_</li><li>`combustion` _(Contains throttle mode with a gas engine-powered motor)_</li></li>`combustion_diesel` _(Contains throttle mode with a diesel engine-powered motor)_</li><li>`hyrbid` _(Contains throttle mode with a mixed gas engine-powered and battery-powered motor)_</li><li>`hybrid_diesel` _(Contains throttle mode with a mixed diesel engine-powered and battery-powered motor)_<br /><br />This field was inspired by, but differs from the propulsion types field described in the [Open Mobility Foundation Mobility Data Specification](https://github.com/openmobilityfoundation/mobility-data-specification/blob/master/provider/README.md#propulsion-types).
```

Comment:
I'm not sure if we're too specific about cars here, again. To future-proof, I think it would be ideal to have just `hybrid` and `combustion` here and then create a fuel-type enum (`diesel`, `electric`, `gasoline`, `LPT`, `E10`, ...)?